File: unix/boot/spp/rpp/ratlibr/addset.r
include	defs

# addset - put c in string (j) if it fits, increment j

   integer function addset (c, str, j, maxsiz)
   integer j, maxsiz
   character c, str (maxsiz)

   if (j > maxsiz)
      addset = NO
   else {
      str(j) = c
      j = j + 1
      addset = YES
      }

   return
   end

File: unix/boot/spp/rpp/ratlibf/addset.f
      integer function addset (c, str, j, maxsiz)
      integer j, maxsiz
      integer c, str (maxsiz)
      if (.not.(j .gt. maxsiz))goto 23000
      addset = 0
      goto 23001
23000 continue
      str(j) = c
      j = j + 1
      addset = 1
23001 continue
      return
      end
